src/types/internal.ts

```diff
@@ -201,3 +201,23 @@ export interface ProcedureMethod<MethodArgs, ReturnValue> {
   (args: MethodArgs): Promise<TransactionQueue<ReturnValue>>;
   checkAuthorization: (args: MethodArgs) => Promise<ProcedureAuthorizationStatus>;
 }
+
+export enum HardcodedProtocolOp {
```

Comment:
This enum is not necessary. The values already exist as TxTags, no need to redefine them.

Create an `protocolOpTags` array inside the conversion function and just see if the passed TxTag is in the array